api/handlers/space.go

```diff
@@ -83,7 +86,20 @@ func (h *Space) list(r *http.Request) (*routing.Response, error) {
 		return nil, apierrors.LogAndReturn(logger, err, "Failed to fetch spaces")
 	}
 
-	return routing.NewResponse(http.StatusOK).WithBody(presenter.ForList(presenter.ForSpace, spaces, h.apiBaseURL, *r.URL)), nil
+	var orgs []repositories.OrgRecord
+	if spaceList.Include != "" && len(spaces) > 0 {
```

Comment:
We already have a somewhat generic pattern for resolving includes from the v3 api. There is an include resolver, here is an example how it is used for service offerings https://github.com/cloudfoundry/korifi/blob/11456cc4c7b7e7f77dbbc00f7c6b3f900957d445/api/handlers/service_offering.go#L74

Here is a PR that got merged recently that demonstrates well what needs to be done in order to handle a new include: https://github.com/cloudfoundry/korifi/pull/3893/files

I would recommend that we follow the same pattern for space includes